pkg/bootstrap/docker/status.go

```diff
@@ -0,0 +1,136 @@
+package docker
+
+import (
+	"fmt"
+	"io"
+	"strings"
+	"time"
+
+	units "github.com/docker/go-units"
+	docker "github.com/fsouza/go-dockerclient"
+	"github.com/spf13/cobra"
+
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+
+	"github.com/openshift/origin/pkg/bootstrap/docker/dockerhelper"
+	"github.com/openshift/origin/pkg/bootstrap/docker/errors"
+	"github.com/openshift/origin/pkg/bootstrap/docker/openshift"
+	"github.com/openshift/origin/pkg/cmd/server/api"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+)
+
+const (
+	// CmdStatusRecommendedName is the recommended command name
+	CmdStatusRecommendedName = "status"
+
+	cmdStatusLong = `
+Show the status of the local OpenShift cluster.
+
+If you started your OpenShift with a specific docker-machine, you need to specify the
+same machine using the --docker-machine argument.
+`
+	cmdStatusExample = `
+	# See status of local OpenShift cluster
+  %[1]s
+
+  # See status of OpenShift cluster running on Docker machine 'mymachine'
+  %[1]s --docker-machine=mymachine
+`
+)
+
+// NewCmdStatus implements the OpenShift cluster status command.
+func NewCmdStatus(name, fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
+	config := &ClientStatusConfig{}
+	cmd := &cobra.Command{
+		Use:     name,
+		Short:   "Show OpenShift on Docker status",
+		Long:    cmdStatusLong,
+		Example: fmt.Sprintf(cmdStatusExample, fullName),
+		Run: func(c *cobra.Command, args []string) {
+			kcmdutil.CheckErr(config.Status(f, out))
```

Comment:
we should reuse the code to print the error that we have here:
https://github.com/openshift/origin/blob/master/pkg/bootstrap/docker/printer.go#L71
(otherwise additional error details like the cause don't get printed out)